web/modules/custom/sfgov_vaccine/src/Form/FilterSitesForm.php

```diff
@@ -71,39 +71,50 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     $form['container']['toggle']['items'] = [
       '#type' => 'container',
+      '#attributes' => [
+        'class' => [
+        ]
+      ]
     ];
 
     // Single checkboxes.
-    $form['container']['toggle']['items']['single_checkboxes'] = [
+    $form['container']['toggle']['items']['access'] = [
       '#type' => 'container',
+      '#attributes' => [
+        'class' => [
+          'mb-40'
+        ],
+      ],
     ];
 
-    // Single checkboxes - kids 5 to 11
-    $form['container']['toggle']['items']['single_checkboxes']['kids5to11'] = [
-      '#type' => 'checkbox',
-      '#title' => $this->t($this->settings('form_strings.kids5to11')),
-      '#default_value' => FALSE,
-    ];
+    $pediatric_options = [];
+    $option_values = $this->settings('form_strings.pediatric.options');
```

Comment:
```suggestion
    $option_values = $this->t($this->settings('form_strings.pediatric.options'));
```

might have this syntax wrong 😬

Reply:
ah i see this should be happening on line 93 below...however I'm still seeing this
![Uploading Screen Shot 2022-06-17 at 12.55.56 PM.png…]()